solutions/USACO jan19-mountain-view.mdx

```diff
@@ -0,0 +1,75 @@
+---
+id: usaco-896
+source: USACO Silver 2019 January
+title: Mountain View
+author: Jesse Choe
+---
+
+[Official Analysis](http://usaco.org/current/data/sol_prob3_silver_dec20.html)
```

Comment:
wrong editorial lol

Reply:
yeet oops. i kinda used the dec20 one as a "template".